src/utils/generateUniqueToken.ts

```diff
@@ -1,3 +1,4 @@
+const crypto = require("crypto");
```

Comment:
We should not require crypto as it is available globally from node 18

Reply:
without this line it was giving below error 
![image](https://github.com/Real-Dev-Squad/discord-slash-commands/assets/61110378/001f3980-8a02-434a-b32c-05489c211544)

Reply:
but now te issue is, this require is not allowed but when I am using import it gives the same error:/